mod/web_services/classes/ElggApiKey.php

```diff
@@ -119,4 +119,36 @@ public function regenerateKeys() {
 		
 		return true;
 	}
+	
+	/**
+	 * Check is the API key is active
+	 *
+	 * @return bool
+	 */
+	public function isKeyActive() {
```

Comment:
change function name to `hasActiveKeys` to keep plurar usage of the keys word intact